napari/components/_viewer_key_bindings.py

```diff
@@ -24,40 +24,12 @@ def toggle_ndisplay(viewer):
         viewer.dims.ndisplay = 3
 
 
-@ViewerModel.bind_key('Left')
-def increment_dims_left(viewer):
-    """Increment dimensions slider to the left."""
-    viewer.dims._increment_dims_left()
-
-
-@ViewerModel.bind_key('Right')
-def increment_dims_right(viewer):
-    """Increment dimensions slider to the right."""
-    viewer.dims._increment_dims_right()
-
-
-@ViewerModel.bind_key('Alt-Up')
-def focus_axes_up(viewer):
-    """Move focus of dimensions slider up."""
-    viewer.dims._focus_up()
-
-
-@ViewerModel.bind_key('Alt-Down')
-def focus_axes_down(viewer):
-    """Move focus of dimensions slider down."""
-    viewer.dims._focus_down()
-
-
-@ViewerModel.bind_key('Control-E')
-def roll_axes(viewer):
-    """Change order of the visible axes, e.g. [0, 1, 2] -> [2, 0, 1]."""
-    viewer.dims._roll()
-
-
-@ViewerModel.bind_key('Control-T')
-def transpose_axes(viewer):
-    """Transpose order of the last two visible axes, e.g. [0, 1] -> [1, 0]."""
-    viewer.dims._transpose()
+ViewerModel.bind_key('Left')(ViewerModel.dims._increment_dims_left)
+ViewerModel.bind_key('Right')(ViewerModel.dims._increment_dims_right)
+ViewerModel.bind_key('Alt-Up')(ViewerModel.dims._focus_up)
+ViewerModel.bind_key('Alt-Down')(ViewerModel.dims._focus_down)
+ViewerModel.bind_key('Control-E')(ViewerModel.dims._roll)
+ViewerModel.bind_key('Control-T')(ViewerModel.dims._transpose)
```

Comment:
Oh, those wont' work as is, as ViewerModel does not have dims yet... 
So likely will need to create methods on ViewerModel.

Reply:
If we changed how we bound keys we could bind these to the dims instead as the callbacks just need the dims, they don't need the full viewer.